tests/pass-dep/tokio/file-io.rs

```diff
@@ -21,6 +21,7 @@ async fn test_create_and_write() -> io::Result<()> {
 
     // Write 10 bytes to the file.
     file.write_all(b"some bytes").await?;
+    file.sync_all().await?; // tokio doesn't necessarily complete writes until you sync.
```

Comment:
```suggestion
    file.flush().await?; // tokio doesn't necessarily complete writes until you sync.
```

Reply:
I won't change it now to speed up the merge of this, but good to know

Reply:
I think we should also change the comment, so we might as well also use `flush` here.
```
// For tokio's file I/O, `await` does not mean "await", so we have to wait some more to make sure it is completed
```

Reply:
I'll just handle this.